src/runtime_src/core/pcie/common/CMakeLists.txt

```diff
@@ -3,6 +3,7 @@
 # Copyright (C) 2022 Advanced Micro Devices, Inc. - All rights reserved
 add_library(core_pciecommon_objects OBJECT
   aie_stubs.cpp
+  asd_parser.cpp
```

Comment:
I am not confident that asd_parser belongs in this directory. Shouldn't it be in src/runtime_src/core/common with the other parsers? Like xclbin_parser? This seems like something that should be in the xrt_coreutil library rather than the xrt_core library.

Reply:
Hi @dbenusov-xilinx, my thoughts were that it is used for parsing aie status on pcie platforms only so I have added it in pcie common area. On edge the parser is different as in those platforms we get data using sysfs.
Let me move it common space as you are right that it should belong to xrt_coreutil.
It is named as asd_parser as it parses aie status dump information.